src/molecules/Toggle/index.jsx

```diff
@@ -0,0 +1,45 @@
+export class Toggle extends React.Component{
+  constructor(props) {
+    super(props);
+    this.state = {
+      isActive: false
+    };
+    this.toggleSwitch = this.toggleSwitch.bind(this);
+  };
+
+  componentDidMount(){
+    const {
+      onChange
+    } = this.props
+  };
+
+  toggleSwitch(){
+    this.state.isActive ? this.setState({ isActive: false }) : this.setState({ isActive: true});
+    if(this.props.onChange){
+      this.props.onChange
```

Comment:
Dropped semi-colon here would get you bullied by a linter, and also I think you will want it to read:
`this.props.onChange()` to call the function.

Also: Can you provide an example that showcases this functionality working on the `.example.mdx` page?